File: Go_api/Models/order.go
package models

import (
	"gorm.io/gorm"
)

type Order struct {
	gorm.Model
	UserID      uint
	User        User `gorm:"foreignKey:UserID"`
	Status      string
	TotalAmount float64
	PaymentID   uint
	Payment     Payment `gorm:"foreignKey:PaymentID"`
	Items       []OrderItem
}

File: Django_api/backend/models/order.py
from django.db import models
from django.conf import settings
from .payment import Payment

class Order(models.Model):
    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
    status = models.CharField(max_length=255)
    total_amount = models.FloatField()
    payment = models.ForeignKey(Payment, on_delete=models.CASCADE)